src/components/attendee/NavOverlay/ProfileOverlay/BasicInfo/ProfileModalAvatar/ProfileModalAvatar.module.scss

```diff
@@ -0,0 +1,22 @@
+.ProfileModalAvatar {
```

Comment:
Same comment as above and all the following copies from older SCSS files: we can  remove the top enclosing class and replace those `&__` with a simple dot `.` in most of the cases

Reply:
[04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)